code/Makefile

```diff
@@ -20,5 +20,6 @@ SEARCH  := ./headers ./src
 # set REATIVE to the next higher directory 
 # and defer to the Makefile.due there
 RELATIVE := $(RELATIVE)../
+RESULTS += main.lss main.lst
```

Comment:
Is this still necessary for master?

Reply:
I think this is really handy. As this will be only used when compiling the main of this library I don't think this needs to be removed.